scripts/solution2.sh

```diff
@@ -0,0 +1 @@
+mkdir -p html && echo -e "<h1>Proffessor Index</h1>\n<ul>" > html/index.html && grep Prof names.txt | while read -r line; do echo "<li><a href="./$(echo "$line" | awk '{print $2 $3}').html">$line</a></li>" >> html/index.html; done && echo "</ul>" >> html/index.html
```

Comment:
I cant lie, these are quite complex. I think the work is excellent, the challenges are involved and will provide a great learning opportunity but we should label them as advanced challenges (particularly the second).

Reply:
Ya, I can see that, but kinda means need to come up with some basic challenges then also.